kuma/static/js/wiki-edit.js

```diff
@@ -544,62 +554,83 @@
         });
     }
 
+    // update the UI to reflect status of draft
     function updateDraftState(action) {
         $draftButton.show();
         var now = new Date();
         var nowString = now.toLocaleDateString() + ' ' + now.toLocaleTimeString();
 
+        // delate old status
         $draftStatus.empty();
+
         if(action){
+            // clone elements and populate
             var $updateAction = $draftAction.clone().text(action);
             var $updateTime = $draftTime.clone().attr('title', now.toISOString()).text(nowString);
-            $draftStatus.append(gettext('draft') + ' ').append($updateAction).append(' ').append($updateTime);
+            // append elements
+            $draftStatus.append(gettext('Draft') + ' ').append($updateAction).append(' ').append($updateTime);
```

Comment:
Yeah, I'm sure it does and I'm not happy with it but this is how it was done before and I didn't want to re-think it as part of this.